tests/functional/conftest.py

```diff
@@ -31,7 +31,7 @@ async def _product_with_full_access():
     await _set_default_rate_limit(product)
     await product.update_scopes([
         "personal-demographics-service:USER-RESTRICTED",
-        "urn:nhsd:apim:app:level3:",
+        "urn:nhsd:apim:app:level3:personal-demographics-service",
```

Comment:
Does this make a difference? Is it more realistic?

Reply:
Yeah, its exactly what we define for the PDS scopes. So yeah more realistic